include/dem/distributions.h

```diff
@@ -97,6 +102,80 @@ class NormalDistribution : public Distribution
    * @brief Random number generator for the diameter selection.
    */
   std::mt19937 gen;
+
+  /**
+   * @brief Cut off diameters values.
+   */
+  double dia_min_cutoff, dia_max_cutoff;
```

Comment:
I would comment the two parameters indepedently so that the doxygen documentation works correctly.